notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-wav2vec2.ipynb

```diff
@@ -8,12 +8,12 @@
    },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    predicted_text = MetricWER.decode_logits(predictions)
</pre><p>it is slightly strange that we use metric for postprocessing. Can we split were calculation and postprocessing step?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1059/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>